.github/autobump.txt

```diff
@@ -341,6 +341,7 @@ clingo
 clipboard
 clojurescript
 closure-compiler
+closure-lsp
```

Comment:
actually a typo, should be `clojure-lsp`

fix in https://github.com/Homebrew/homebrew-core/pull/170248